### PR TITLE
Revised fuzzy chains from issue #2345

### PR DESCRIPTION
**Pull request by fernandotellado**
#2345

### CODE REVIEW

File: languages/affiliate-wp-es_ES.po [hunk 1]
```diff
@@ -6319,37 +6302,35 @@ msgstr "Directo"
 #~ "username) and choose whether or not to show <strong>Pretty Affiliate "
 #~ "URLs</strong> to your affiliates."
 #~ msgstr ""
-#~ "Sus afiliados ya pueden promover una gran variedad de \n"
+#~ "Tuss afiliados ya pueden promover tu web usando una gran variedad de \n"
```
Comment (michaelbeil) [inline languages/affiliate-wp-es_ES.po]: Tus?

Comment (fernandotellado) [reply languages/affiliate-wp-es_ES.po]: Ooops, a typing error

Comment (fernandotellado) [reply languages/affiliate-wp-es_ES.po]: Yeah, it's Tus not Tuss

Comment (fernandotellado) [reply languages/affiliate-wp-es_ES.po]: May yo please change it? I'm at this moment with the mobile and cannot change it safely

Comment (michaelbeil) [reply languages/affiliate-wp-es_ES.po]: Vuestros?

Comment (fernandotellado) [reply languages/affiliate-wp-es_ES.po]: Nope, is Tus

**Pull request merged**